test/functional/test.run.js

```diff
@@ -288,7 +276,6 @@ describe("jpm run", function () {
       }
 
       it("run with only -v option", function(done) {
-        process.chdir(paramDumpPath);
```

Comment:
after this, does the cmd need `{ addonDir: paramDumpPath ...` ? I can't quite tell what it's testing. I also noticed some `chdir(paramDumpPath)` calls are still in tests right after this but that's probably fine.